tests/test_tw_surface.rs

```diff
@@ -21,7 +21,7 @@ use r3bl_rs_utils::*;
 fn test_simple_2_col_layout() -> CommonResult<()> {
```

Comment:
This file is missing an import statement. Add the following:
`use crossterm::style::Color;`